user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md

```diff
@@ -21,13 +16,89 @@ Advantages of DataMiner Storage as a Service (STaaS) include:
 
 - Outstanding performance, scalability, and redundancy
 
-- Compatibility with all data types
-
 - Cost-effectiveness
 
 - Safe and secure data
 
 > [!TIP]
->
-> - To read more about Storage as a Service and its advantages, see [DataMiner STaaS: a game-changer for your storage needs](https://community.dataminer.services/dataminer-staas-a-game-changer-for-your-storage-needs/).
-> - As Storage as a Service is **scheduled for production in Q3 of 2023**, we are working on adding more information to this section. Check back for updates frequently!
+> If you have any questions or need support with the STaaS feature, contact <staas@dataminer.services>.
+
+## Setting up STaaS
+
+1. [Upgrade your DataMiner System](xref:Upgrading_a_DataMiner_Agent) to version 10.3.10 [CU1] or higher.
+
+   > [!IMPORTANT]
+   > We recommend always upgrading DataMiner to the latest available version to get the latest features and performance updates.
+
+1. Make sure your DataMiner System is [connected to dataminer.services](xref:Connecting_your_DataMiner_System_to_the_cloud).
+
+1. Make sure you have at least **DataMiner CloudGateway 2.8.0** installed on the system. See [Upgrading nodes to the latest DxM versions](xref:Managing_cloud-connected_nodes#upgrading-nodes-to-the-latest-dxm-versions).
+
+1. Contact your Skyline representative or <staas@dataminer.services> to register you system to use STaaS.
+
+1. Wait until you receive confirmation that the registration is completed.
+
+1. **Optionally**, contact your Skyline representative or <staas@dataminer.services> to migrate your existing data to STaaS.
+
+1. On each DataMiner Agent in the cluster, in the `C:\Skyline DataMiner` folder, open *DB.xml* and edit it to look like this:
+
+   ```xml
+   <?xml version="1.0"?>
+   <DataBases xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns="http://www.skyline.be/config/db">
+      <DataBase active="true" local="true" search="true" cloud="true" type="CloudStorage"/>
+   </DataBases>
+   ```
+
+1. Restart DataMiner to begin using STaaS.
+
+## Limitations
+
+To **migrate existing data** to STaaS, the following limitations apply:
+
+- If you migrate from a Cassandra database, logger tables cannot be migrated yet. Support for the migration of logger tables will be introduced in a future DataMiner version.
+- Migration of an Elasticsearch database that is defined in [DBConfiguration.xml](xref:DBConfiguration_xml) instead of [DB.xml](xref:DB_xml) (e.g. because a setup with multiple Elasticsearch clusters is used) is not yet supported.
+- Migration from a MySQL setup is not yet supported.
+
+In addition, the following **other limitations** currently apply:
+
+- [Jobs](xref:jobs), [Ticketing](xref:ticketing), and [API Deployment](xref:Overview_of_Soft_Launch_Options#apideployment) data are not supported.
+- Indexing engine functionality such as search, suggestions, aliases, and aggregation are not supported.
```

Comment:
Since this also uses the indexing engine, is SRM supported? If not, we'd better also mention this.

Reply:
Only search, suggestions, aliases, and aggregation are not supported. 
with the exception of the above, everything from Elastic works (SRM, DOM, etc... work)